packages/checkbox/src/Checkbox.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { mount, render } from "enzyme";
+import Checkbox from "./Checkbox";
+
+describe("Checkbox", () => {
+  const subject = (props = {}) =>
+    mount(<Checkbox title="Checkbox" {...props} />);
+  let wrapper;
+  let checkbox;
+
+  describe("integration", () => {
```

Comment:
_Future_: As we discussed before, these tests will all fail once we convert the component to CSS-in-JS. Additionally, these kinds of tests are better suited for snapshots, since they can be verified and updated much more easily. With that said, I won't ask for a rewrite now since that will have to happen later anyway.

Reply:
No problem, we subbed this out for a basic snapshot (ala Dropdown) in b425df7d